Doc/library/pyexpat.rst

```diff
@@ -231,6 +231,54 @@ XMLParser Objects
    .. versionadded:: 3.13
 
 
+:class:`xmlparser` objects have the following methods to mitigate some
+common XML vulnerabilities.
+
+.. method:: xmlparser.SetAllocTrackerActivationThreshold(threshold, /)
```

Comment:
For new methods, should we use snake_case naming?

Also, are there corresponding getters/properties to query the current values?

A